wiki/Skinning/Compendium/en.md

```diff
@@ -0,0 +1,269 @@
+[o!s]: /wiki/shared/mode/osu.png "osu!standard"
+[o!t]: /wiki/shared/mode/taiko.png "osu!taiko"
+[o!c]: /wiki/shared/mode/catch.png "osu!catch"
+[o!m]: /wiki/shared/mode/mania.png "osu!mania"
+
+# Skins Compendium
```

Comment:
Just `Compendium` is enough
Your current one will look like this:
![image](https://user-images.githubusercontent.com/14264992/36161695-f1e2abcc-10a9-11e8-9af9-13b86e5f7ec0.png)
instead of this:
![image](https://user-images.githubusercontent.com/14264992/36161672-e6489308-10a9-11e8-9ecb-05221d61871f.png)

----

either that or just rename the folder to `Skins_Compedium`